leetcodePython/Heap/heap_502.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+from heapq import *
+
+
+class Solution:
+    def findMaximizedCapital(self, k: int, w: int, profits: List[int], capital: List[int]) -> int:
+        n = len(profits)
+        projects = list(zip(capital, profits))
+        projects.sort()
+        heap = []
+        ptr = 0
+        capital = w
+        for i in range(k):
+            while ptr < n and projects[ptr][0] <= capital:
+                heappush(heap, -projects[ptr][1])
+                ptr += 1
```

Comment:
In every iteration, base on our current capital:
1. Put the capital affordable project profit in heap (sorted)